src/SamplesApp/SamplesApp.UITests/UnoSamples_Tests.Keyboard.cs

```diff
@@ -365,6 +365,21 @@ public void TextBox_BeforeTextChanging()
 			Assert.AreEqual(Final2, GetText(textBlock));
 		}
 
+		[Test]
```

Comment:
`[AutoRetry]` ? ;-)

Reply:
Why?

Reply:
It’s now a requirement, tests will fail if the attribute is not present.

Reply:
...why??

Reply:
Because tests on CI are not stable enough and it's a way to minimize broken builds.

Reply:
Sure but it doesn't make much sense to have an attribute in this scenario - just retry all the tests.